cmd/events-main.go

```diff
@@ -53,7 +48,7 @@ COMMANDS:
 }
 
 // mainEvents is the handle for "mc events" command.
-func mainEvents(ctx *cli.Context) {
+func mainEvents(ctx *cli.Context) error {
```

Comment:
this function returns nil always, what is the thought behind this way of doing things? several other functions also are behaving the same way, so wondering.